mobile/function/map_page_function.js

```diff
@@ -26,11 +27,12 @@ const getMapPage = (routeId) => {
     //Fetching the data from the api.
     const fetchData = useCallback(async () => {
         const response = await fetch(
-            "http://10.232.15.11:3000/api/routes/route/" + routeId
+            "http://" + IP + ":3000/api/routes/route/" + routeId
```

Comment:
Ik zou hier van maken `${API}/api/routes/route/${routeId}`
Op deze manier is het in mijn ogen leesbaarder. Ook zou ik in plaats van alleen het IP het hele adres in de env zetten.
Dus dat IP dan "http://10.232.15.11:3000" is